examples/tutorials/hindi/Part 01 - The Basic Tools of Private Deep Learning.ipynb

```diff
@@ -0,0 +1,965 @@
+   "metadata": {},
+   "source": [
+    "उस मेटाडेटा (Metadata) की जाँच करें!\n",
+    "\n",
+    "संकेत करने के लिए विशिष्ट दो मुख्य विशेषताएं हैं:\n",
+    "\n",
+    "- `x_ptr.location : bob`, स्थान, उस स्थान का संदर्भ जिसे सूचक (Pointer) इंगित कर रहा है\n",
+    "- `x_ptr.id_at_location : <random integer>`, वह आईडी जहां स्थान पर Tensor संग्रहीत होता है\n",
+    "\n",
+    "वे प्रारूप में मुद्रित होते हैं `<id_at_location>@<location>`\n",
+    "\n",
+    "अन्य अधिक सामान्य विशेषताएँ भी हैं:\n",
+    "- `x_ptr.id : <random integer>`, हमारे पॉइंटर Tensor की आईडी, इसे बेतरतीब ढंग (Randomly Allocated) से आवंटित किया गया था\n",
+    "- `x_ptr.owner : \"me\"`, वर्कर जो पॉइंटर (Pointer) टेनर का मालिक है, यहाँ उसका स्थानीय कर्मी है, जिसका नाम \"me\" है\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<VirtualWorker id:bob #tensors:3>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x_ptr.location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<VirtualWorker id:bob #tensors:3>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bob"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bob == x_ptr.location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "14597461368"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x_ptr.id_at_location"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<VirtualWorker id:me #tensors:0>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x_ptr.owner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "आपको आश्चर्य हो सकता है कि स्थानीय कर्मचारी जो सूचक (Pointer) का मालिक है, वह भी एक VirtualWorker है, हालांकि हमने इसे नहीं बनाया है।\n",
+    "मजेदार तथ्य, ठीक उसी तरह जैसे बॉब (Bob) के लिए हमारे पास VirtualWorker ऑब्जेक्ट था, हम (डिफ़ॉल्ट रूप से) हमेशा हमारे लिए एक होते हैं। जब हम `hook = sy.TorchHook ()` कहते हैं, तो यह कार्यकर्ता स्वचालित रूप से बनाया जाता है और इसलिए आपको आमतौर पर इसे स्वयं बनाने की आवश्यकता नहीं है।"
```

Comment:
"हम (डिफ़ॉल्ट रूप से) हमेशा हमारे लिए एक होते हैं" does not seem to have the same meaning as "we (by default) always have one for us as well"